spec/factories/prescription_drugs.rb

```diff
@@ -8,6 +8,8 @@
     rxnorm_code { Faker::Code.npi }
     device_created_at { Time.current }
     device_updated_at { Time.current }
+    frequency { "OD" }
+    duration_in_days { 10 }
```

Comment:
I would keep the regular factory _without_ these, and introduce a `for_teleconsultation` trait that adds them (as well as a teleconsultation ID, depending on which PRs get merged first.)